### PR TITLE
Accept my merge request!!!

### PR DESCRIPTION
**Pull request by KatamariJr**
blah blah

### CODE REVIEW

**Comment by tom-helfrich** [comment]
Check our Code Design Pattern patterns; you should use functional components instead of class components.

**Comment by tom-helfrich** [comment]
Test test

**Comment by KatamariJr** [comment]
i fixed it!

File: src/main/java/com/cmsc495/spacesim/model/Model.java [hunk 1]
```diff
@@ -5,6 +5,7 @@
  */
 package com.cmsc495.spacesim.model;
 
+//comment
```
Comment (tom-helfrich) [inline src/main/java/com/cmsc495/spacesim/model/Model.java:8]: //comment is a vague comment

**Review by tom-helfrich — CHANGES_REQUESTED** [review]

**Review by tom-helfrich — APPROVED** [review]

**Pull request merged**